orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -119,6 +119,29 @@ def __init__(self, module):
         self._save_onnx = False
         self._save_onnx_prefix = ''
 
+        # CPP extension to get torch CUDA allocator's alloc and free function addresses
+        self._use_external_cuda_allocator = True
+        if self._use_external_cuda_allocator:
+            self._torch_cuda_allocator = self._load_torch_allocator_cpp_extension()
+
+    def _load_torch_allocator_cpp_extension(self):
```

Comment:
Note that `self` is not used in the implementation.
As I mentioned in a previous review, this means this code shouldn't be a method of `ORTModule` but rather a floating function. A similar example is `_create_iobinding`